docs/source/online_dpo_trainer.md

```diff
@@ -29,14 +29,13 @@ from transformers import (
 )
 NUM_DUMMY_SAMPLES = 100
 
-tokenizer = AutoTokenizer.from_pretrained("HuggingFaceTB/SmolLM-135M-Instruct")
-tokenizer.add_special_tokens({"pad_token": "[PAD]"})
+tokenizer = AutoTokenizer.from_pretrained("Qwen/Qwen2-0.5B-Instruct")
 # The model to optimise
-model = AutoModelForCausalLM.from_pretrained("HuggingFaceTB/SmolLM-135M-Instruct")
```

Comment:
The SmolLM models are broken with reward modelling - basically unable to learn anything past random. I suspect it's an issue with the padding token == EOS token, but I'll investigate in a separate PR